### PR TITLE
Css update

### PR DESCRIPTION
**Pull request by novaraptur**
# Description

Update CSS to be responsive; get minimum viable styling.

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [X] Refactor

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [X] Console Dev Tools
- [ ] Cypress

### CODE REVIEW

File: src/Components/Answer/Answer.css [hunk 1]
```diff
@@ -0,0 +1,12 @@
+.answer-button {
+  font-size: 1.5em;
+  height: 200px;
+  width: 200px;
+  margin: 3%;
+}
+
+@media screen and (max-width: 1000px) {
+  .answer-button {
+    width: 25%;
```
Comment (hoomberto) [inline src/Components/Answer/Answer.css:10]: Nice dynamic sizing for this!

File: src/Components/GameBoard/GameBoard.css [hunk 2]
```diff
@@ -1,18 +1,19 @@
...
+  grid-template-columns: 1fr 1fr 1fr 1fr 1fr;
+  width: 100%;
 }
 
-/* .tiles {
-  transform: rotate(90deg);
-} */
-
-@media screen and (max-width: 450px) {
+@media screen and (max-width: 1350px) {
```
Comment (hoomberto) [inline src/Components/GameBoard/GameBoard.css:14]: I appreciate the addition of all these media queries, nice and responsive 🙌

File: src/Components/PastGames/PastGames.js [hunk 3]
```diff
@@ -1,19 +1,15 @@
...
 import { pastGames } from '../../utilities/apiCalls'
 
-class PastGames extends Component {
-  constructor() {
-    super()
-      this.state = {
-        pastGames: []
-      }
-  }
+const PastGames = () => {
```
Comment (hoomberto) [inline src/Components/PastGames/PastGames.js:5]: This looks much more clean with hooks 👍

**Review by hoomberto — APPROVED** [review]

File: src/Components/Game/Game.js [hunk 4]
```diff
@@ -127,7 +127,7 @@ const Game = () => {
     evaluateChoice(choice)
     updateQuestions()
     checkIfOver();
-    setTimeout(letUserPickNext, 10000);
+    setTimeout(letUserPickNext, 100);
```
Comment (shawnmcmahon) [inline src/Components/Game/Game.js]: Whoops my bad!!

**Pull request merged**